pages/admin/profile/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import { Button, Row, Col } from 'antd';
+import Layout from '../../../components/layout';
+import { components } from '../../../components/profile';
+import './index.scss';
+
+const { ViewProfile, EditProfile } = components;
+
+class Profile extends Component {
+    state = { action: 'view' }
+
+    changeAction = () => {
+        const { action } = this.state;
+
+        if (action === 'view') {
+            return this.setState({ action: 'edit' });
+        }
+
+        return this.setState({ action: 'view' });
+    }
+
+    render() {
+        const { action } = this.state;
+        return (
```

Comment:
define a variable` isViewAction = action === 'view'` and use that in the render function